src/sidebar/services/frame-sync.ts

```diff
@@ -190,16 +198,19 @@ export class FrameSyncService {
       this._pendingAnchorStatusUpdates.clear();
     }, 10);
 
+    this._sidebarIsOpen = true;
```

Comment:
This default state might not be correct. I would default this to false and then rely on the host to send a `sidebarOpened` message when the sidebar is opened. You will need to check/ensure that this message is sent by src/annotator/sidebar.ts if the sidebar starts opened. If the host sends this message before the sidebar has loaded that is fine, since `MessagePort` queues any received messages until the receiver is ready to process them.

Reply:
Just checked it. The `sidebarOpened` message is sent at the very beginning when the sidebar is opened, so it does make sense to initialize it as `false`.